custom_components/aquarea/definitions.py

```diff
@@ -880,8 +879,7 @@ def build_binary_sensors(
             heishamon_topic_id="TOP68",
             key=f"{mqtt_prefix}main/Force_Heater_State",
             name="Aquarea Force heater status",
-            state=bit_to_bool,
-            device_class=BinarySensorDeviceClass.RUNNING,
```

Comment:
I'm not sure to understand why this device class would not fit. The doc from https://github.com/Egyras/HeishaMon/blob/master/MQTT-Topics.md says it corresponds to "inactive/active" state which running seems to represents.

Reply:
I think that on/off is better in this case than running/not running.
Running/not running is fine for a water pump.
But maybe I'm wrong.😀
Maybe need leave that one out, now when I thinking.🙄

But the second one, is not questionable

Reply:
lets merge like this